File: CPP/Program/laporanSampah.cpp
#include <iostream>
#include <string>

using namespace std;

class LaporanSampah {
    // Atribut private
    private:
        string idLaporanSampah;
        string totalTransaksi; 
        string totalBerat;

    public:
        // Constructor kosong
        LaporanSampah() {
        }

        // Constructor
        LaporanSampah(string idLaporanSampah, string totalTransaksi, string totalBerat) {
            this->idLaporanSampah = idLaporanSampah;
            this->totalTransaksi = totalTransaksi;
            this->totalBerat = totalBerat;
        }

        // getter setter id lapora sampah
        string getIdLaporanSampah() {return idLaporanSampah;}
        
        void setIdLaporanSampah(string idLaporanSampah) {this->idLaporanSampah = idLaporanSampah;}
        
        // getter setter total transaksi
        string getTotalTransaksi() {return totalTransaksi;}
        
        void setTotalTransaksi(string totalTransaksi) {this->totalTransaksi = totalTransaksi;}
        
        // getter setter total berat
        string getTotalBerat() {return totalBerat;}
        
        void setTotalBerat(string totalBerat) {this->totalBerat = totalBerat;}
        
    // Destructor
    ~LaporanSampah() {    
    }
};

File: JAVA (BONUS)/LaporanSampah.java
public class LaporanSampah {
    // Atribut private
    private String idLaporanSampah;
    private String totalTransaksi;
    private String totalBerat;

    // Constructor kosong
    public LaporanSampah() {
    }

    // Constructor
    public LaporanSampah(String idLaporanSampah, String totalTransaksi, String totalBerat) {
        this.idLaporanSampah = idLaporanSampah;
        this.totalTransaksi = totalTransaksi;
        this.totalBerat = totalBerat;
    }

    // Getter Setter idLaporanSampah
    public String getIdLaporanSampah() {
        return idLaporanSampah;
    }

    public void setIdLaporanSampah(String idLaporanSampah) {
        this.idLaporanSampah = idLaporanSampah;
    }

    // Getter Setter totalTransaksi
    public String getTotalTransaksi() {
        return totalTransaksi;
    }

    public void setTotalTransaksi(String totalTransaksi) {
        this.totalTransaksi = totalTransaksi;
    }

    // Getter Setter totalBerat
    public String getTotalBerat() {
        return totalBerat;
    }

    public void setTotalBerat(String totalBerat) {
        this.totalBerat = totalBerat;
    }
}
